### PR TITLE
Add Jekyll!

### LINKED ISSUE #42: 🖼️ Framework Request: Jekyll
| Question                                   | ✅ Yes | ❌ No |
|--------------------------------------------|-------|------|
| Is this framework already being requested? | []    | [x]   |

### What is the name of the framework?

Jekyll

### What language does the framework use?

HTML / CSS / JS & Some Ruby

### What is the website for the framework?

https://jekyllrb.com/

### PR DESCRIPTION
**Pull request by sampoder**
## Description

Added a basic blog template. do you all want me to use the name icon thing? 

## Related Issue

Closes #42 

## Types of changes

- [ ] New framework or language
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the structure and style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] There is no duplicate pull request for this fix/addition/issue resolution.
- [x] If applicable, my code passes all current tests, and passes new tests that I have written to test it if the current tests do not suffice.

### CODE REVIEW

File: frameworks/jekyll/basic/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# basic
+
+```
+├── _includes
+│   └── head.html
+├── index.html
+├── .DS_Store
```
Comment (cjdenio) [inline frameworks/jekyll/basic/README.md:7]: Is this supposed to be here? Just curious.

Comment (sampoder) [reply frameworks/jekyll/basic/README.md:7]: nope

Comment (sampoder) [reply frameworks/jekyll/basic/README.md:7]: at least the .Ds_store isn't

**Comment by cjdenio** [comment]
You can use this as the badge.
http://img.shields.io/static/v1?label=Framework&message=Jekyll&color=cc0000&logo=jekyll&logoColor=white&style=for-the-badge

File: frameworks/jekyll/basic/README.md (hunk 1 above)
Comment (cjdenio) [reply frameworks/jekyll/basic/README.md]: I wasn't quite sure :joy:

File: README.md [hunk 2]
```diff
@@ -15,3 +15,4 @@
 - ⚛️ [React](frameworks/react)
 - 🌶 [Flask](frameworks/flask)
 - 🦕 [Deno](frameworks/deno)
+[![Jekyll](https://img.shields.io/static/v1?label=Framework&message=Jekyll&color=CC0000&logo=Jekyll&logoColor=white&style=for-the-badge)](frameworks/flask)
```
Comment (cjdenio) [inline README.md]: Please fix the link to point to `frameworks/jekyll`.

Comment (sampoder) [reply README.md]: already fixed in commit b5d8264 lol

**Comment by sampoder** [comment]
Let's wait for all the icons to be updated and then merge this

File: frameworks/jekyll/basic/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# basic
```
Comment (cjdenio) [inline frameworks/jekyll/basic/README.md:1]: One more picky thing: could you make this a 2nd level heading and add the Jekyll badge at the top? Thanks!

File: frameworks/jekyll/basic/README.md (hunk 1 above)
Comment (sampoder) [reply frameworks/jekyll/basic/README.md]: silly macos

File: frameworks/jekyll/basic/README.md (hunk 3 above)
Comment (sampoder) [reply frameworks/jekyll/basic/README.md:1]: i guess, btw i just used @Matt-Gleich 's cli

**Comment by cjdenio** [comment]
Go ahead once #47 is merged.

**Pull request merged**